meetings/2018-08-29-board-meeting.md

```diff
@@ -0,0 +1,43 @@
+# Ruby Together board meeting summary
+
+**August 29, 2018**
+
+**Attendees and Agenda**
+
+Valerie, Jonan, Allison, Adarsh, Coraline, Courtney, and André were present. 
+
+Topics for discussion included:
+
+- Working on the RubyMe OSS mentorship proposal 
+- Reviewing a new proposed granting process
+- Getting new members
+
+## Goodbye Courtney, and New Board Size
+
+Courtney has decided to step down as a board member. We thank her for all her work and service to the organization. 
+
+Based on Courtney’s departure and André’s new role, which includes stepping down from being a board member, the board discussed capping the board's size to five instead of seven. A board consisting of five members is more standard with other nonprofits, and will be easier for coordination purposes.
+
+## [RubyMe](http://rubyme.org)
+
+RubyMe will be announced soon. We’ll be sure to do outreach to make sure folks know about the opportunity.
+
+Blog post here: [https://rubytogether.org/news/2018-09-18-announcing-ruby-me](https://rubytogether.org/news/2018-09-18-announcing-ruby-me) 
+
+## Grant Funding
+
+Grant funding is almost ready to go. All the parts are complete; we just need to check with a lawyer about providing grants.
+
+## Insurance
+
+We’re going to secure liability insurance.
+
+## New Members and Increasing Revenue
+
+Our membership numbers aren’t growing; therefore, our revenue is staying steady or dropping. Allison and Jonan are working together on a subcommittee to brainstorm revenue generating ideas and ideas for increasing membership numbers.
```

Comment:
> Our membership numbers aren’t growing; therefore, our revenue is staying steady or dropping.

Hey @asheren—I'm trying to clarify the above sentence a bit. Can you confirm if either of the following statements are accurate?

- Our revenue is _steadily dropping_.
or
- Our revenue is _currently steady_ but is at risk of dropping in the future.

Thanks!

Reply:
I'm trying to say that our revenue is staying steady but our expenses are growing and therefore we need more members in order to continue supporting our efforts. @indirect can you confirm this statement? Just want to make sure it's correct.

Reply:
@asheren got it! We could probably use that language, tweaking it slightly:

> While our revenue has remained steady, we'll need to attract more members in order to continue supporting our efforts.

I'll connect with @indirect also to get his take on how he'd like this worded!

Reply:
Made a tiny phrasing tweak, but this is great overall. Thanks @asheren and @rubymorillo!